vendor/k8s.io/kubernetes/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -188,6 +188,10 @@ func init() {
 			rbac.NewRule("get", "list", "watch").Groups(storageGroup).Resources("storageclasses").RuleOrDie(),
 			rbac.NewRule("get", "create", "delete").Groups(legacyGroup).Resources("services", "endpoints").RuleOrDie(),
 			rbac.NewRule("get").Groups(legacyGroup).Resources("secrets").RuleOrDie(),
+			// Cinder provisioner needs to get list of nodes in order
```

Comment:
can't we just copy this from upstream: https://github.com/kubernetes/kubernetes/blob/0613ae5077b280eabbff593bc2dc34fe2d03538d/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go#L192

reduces diff in next rebase.